mysql/src/main/scala/zio/sql/mysql/MysqlSqlModule.scala

```diff
@@ -27,12 +26,13 @@ trait MysqlSqlModule extends Sql { self =>
   }
 
   object MysqlFunctionDef {
-    val Crc32     = FunctionDef[String, Long](FunctionName("crc32"))
-    val Degrees   = FunctionDef[Double, Double](FunctionName("degrees"))
-    val Log2      = FunctionDef[Double, Double](FunctionName("log2"))
-    val Log10     = FunctionDef[Double, Double](FunctionName("log10"))
-    val Pi        = Expr.FunctionCall0[Double](FunctionDef[Any, Double](FunctionName("pi")))
-    val BitLength = FunctionDef[String, Int](FunctionName("bit_length"))
+    val Crc32       = FunctionDef[String, Long](FunctionName("crc32"))
+    val Degrees     = FunctionDef[Double, Double](FunctionName("degrees"))
+    val Log2        = FunctionDef[Double, Double](FunctionName("log2"))
+    val Log10       = FunctionDef[Double, Double](FunctionName("log10"))
+    val Pi          = Expr.FunctionCall0[Double](FunctionDef[Any, Double](FunctionName("pi")))
+    val BitLength   = FunctionDef[String, Int](FunctionName("bit_length"))
+    val CurrentTime = Expr.ParenlessFunctionCall0[OffsetTime](FunctionName("current_time"))
```

Comment:
How about considering the alphabetical order here? I see that the `BitLength` broke it a bit. Now one more. 

It's useful when considering the code completion feature and what suggestions the library user sees.

Reply:
Yup makes perfect sense :+1: . fixed